src/frontend/src/styles/main.css

```diff
@@ -269,7 +269,7 @@ hr {
   margin-top: -3rem;
 }
 
-#aboutLink {
+#navbar a {
```

Comment:
This way you're losing the `margin-top` on the navbar (You'll have to put that on `#navbar` directly). I'd also rather target the links with a class, avoiding nested selectors as much as possible.

Reply:
I've just pushed a commit un-nesting the selectors and fixing the margin issue (I've also added some margin at the bottom, now it doesn't look like the buttons just fell to the bottom of the page with gravity). Let me know what you think!